translations/en.json

```diff
@@ -2546,8 +2546,8 @@
     "derivation-path-desc": "Derivation paths are the routes your Status Wallet uses to generate addresses from your private key.",
     "select-networks": "Select networks",
     "generating-keypair": "Generating keypair...",
-    "keypair-name": "Keypair name",
-    "keypair-name-description": "Name keypair for your own personal reference",
+    "keypair-name": "Key pair name",
```

Comment:
please check this with design, as I think it might have been a design bug to add the space :)

Reply:
Checked with design team, it should "Key pair"